rest-servlet/src/main/java/org/hawkular/inventory/rest/RestEnvironments.java

```diff
@@ -57,14 +60,16 @@
 
     @GET
     @Path("/{tenantId}/environments")
-    @ApiOperation("Returns all environments under given tenant.")
+    @ApiOperation("Returns all environments under given tenant. Accepts paging query parameters.")
```

Comment:
> Also, how do you support multiple values passed as via one parameter name (i.e. sort=a&sort=b) using @QueryParam?

`@QueryParam List<Foo> foos`
Agreed, the code wouldn't be as DRY as it is, but it could be done also by some util class like this:

```java

  getSomething(...., 
          @QueryParam("page") String page, 
          @QueryParam("per_page") String perPage, 
          @QueryParam("sort") List<String> sort, 
          @QueryParam("order") String order) {
      Pager pager = RequestUtil.extractPaging(page, perPage, sort, order);
      ...
  }
```